Rakefile

```diff
@@ -125,6 +127,16 @@ task :student do
   Rake::Task["post"].execute
 end
 
+## This requires some configs 
+desc "Package and ship a VM"
+task :ship do
+  vmname = ENV['VMNAME'].split(".").first
+  ovapath=retrieve_ova(vmname)
+  push_vmware(ovapath)
+  ship_vm_to_dir(ovapath, "/mnt/nfs/EducationVMS/#{ENV['VMTYPE']}")
```

Comment:
So this is supposed to be run on a specific machine, yes? So that it ends up in the correct place when you ship the ova?